cql3/statements/delete_statement.cc

```diff
@@ -99,9 +99,6 @@ delete_statement::delete_statement(cf_name name,
     , _deletions(std::move(deletions))
     , _where_clause(std::move(where_clause))
 {
-    if (_attrs->time_to_live) {
```

Comment:
If you removed this, you probably don't need the mask machinary you added in the previous patch.

Reply:
changed to assert in v4